hr_expense_advance_clearing/data/advance_product.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version='1.0' encoding='utf-8'?>
```

Comment:
I don't believe this is needed and can be removed.

Reply:
Done